test/functional/mempool_package_rbf.py

```diff
@@ -169,22 +169,34 @@ def test_package_rbf_additional_fees(self):
 
         self.log.info("Check replacement pays for incremental bandwidth")
         package_hex3, package_txns3 = self.create_simple_package(coin, parent_fee=DEFAULT_FEE, child_fee=DEFAULT_CHILD_FEE)
```

Comment:
Renaming would help distinguish the variables which are currently reassigned later. Also, since fee doesn't matter.

```suggestion
        _, placeholder_txns3 = self.create_simple_package(coin)
```

Reply:
done